scanner/updater/version/RELEASE_VERSION

```diff
@@ -0,0 +1,10 @@
+# This file lists the ACS versions supported by Scanner V4 and Scanner V4 Updater.
+# Since Scanner V4 shares the same version as ACS,
+# Scanner V4 Updater updates vulnerability data for all versions of Scanner V4 listed in this file
+# and uploads that data to Google Storage.
+# Various versions of Scanner V4 matchers will import vulnerability data based on their respective versions from Google Storage.
```

Comment:
nit: Each version of Scanner's vulnerability updater will upload vulnerability data to Google Storage based on its respective version.